packages/@atjson/renderer-html/src/index.ts

```diff
@@ -286,6 +287,30 @@ export default class HTMLRenderer extends Renderer {
     }</div> <div style=" font-size: 15px; line-height: 21px; color: #000000; font-weight: 600; "> View on Threads</div></div></a></blockquote><script async src="https://www.threads.net/embed.js"></script>`;
   }
 
+  *RedditEmbed(embed: Block<RedditEmbed>) {
+    let content = this.getSlice(embed.attributes.content);
+    let attrs = this.htmlAttributes({
+      class: "reddit-embed-bq",
+      style: "height:500px",
+      id: embed.attributes.anchorName,
+      "data-embed-height": embed.attributes.height.toString(),
+      "data-embed-showusername": embed.attributes.hideUsername
+        ? "false"
+        : undefined,
+      "data-embed-showmedia": embed.attributes.hidePostContent
+        ? "false"
+        : undefined,
+      "data-embed-showedits": embed.attributes.hidePostContentIfEditedAfter
+        ? "false"
+        : undefined,
+      "data-embed-created": embed.attributes.hidePostContentIfEditedAfter,
+    });
+
+    return `<blockquote ${attrs.join(" ")}>${
+      content ? this.render(content) : ""
+    }</blockquote><script async src="https://embed.reddit.com/widgets.js" charset="UTF-8"></script>`;
```

Comment:
So I think you mentioned there was an issue with this script where having multiple of these embed snippets on a page caused some sort of ID clash. Is that correct? If so, are we blocked on this, or can we assume that the snippet itself won't change?

Reply:
I don't think we are blocked on this here, I can ensure that the `id` is unique for this renderer but I am not sure that we need to care about it :)